1 HW_Standard/HW_Teddiursa.yaml

```diff
@@ -20,3 +20,26 @@ nature: Jolly
 ivs: 31
 evs: {hp: 0, atk: 136, def: 56, spA: 0, spD: 64, spe: 252}
 hidden: false
+---
+species: Teddiursa
+ingamename: BuildrBear
+displayname: SCP-1048
+setname: Halloween
+tags: [halloween]
+item: Lagging Tail
+ability: Cute Charm
+moves:
+    - [Pluck]
+    - [Thief]
+    - [Screech]
+    - [Growth]
+gender: [m,f]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Heal]
+shiny: True
+nature: Jolly
+ivs: 31
+evs: {hp: 0, atk: 252, def: 0, spA: 0, spD: 0, spe: 252}
```

Comment:
quite weak, i would either remove lagging tail and invest some in bulk, or keep it, switch to adamant, and invest a lot in decent amount in bulk